### PR TITLE
Fill mandatory fields to run validations.

### PR DESCRIPTION
**Pull request by krritik**
Change - 
If the user clicks on the "Validation Button"  before entering mandatory data such as Storage account details, Network share credentials, etc. It does not validation, instead, it throws an error that data in some fields are missing.

### CODE REVIEW

File: extensions/sql-migration/src/wizard/databaseBackupPage.ts [hunk 1]
```diff
@@ -1335,6 +1335,29 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 			this.wizard.message = { text: '' };
 		}
 
+		const errorDetails: string[] = [];
+		if (this._windowsUserAccountText.value === undefined || this._windowsUserAccountText.value === '') {
```
Comment (Ramudaykumar) [inline extensions/sql-migration/src/wizard/databaseBackupPage.ts:1339]: it is good that are adding some validation, but should not we have character set validation. Like Username, Storage Account name and Container name we have defined set of characters. Special correctors are not supported.  You may quickly check with PM.

**Comment by Ramudaykumar** [comment]
Looks good, but still these validations are just for field remains blank. is that sufficient?

**Review by Ramudaykumar — APPROVED** [review]
Need sign-off from PM.

**Comment by krritik** [comment]
> Need sign-off from PM.

@abhims14, once look into this.

**Comment by Ramudaykumar** [comment]
@krritik as you suggested in call that you would take further validation in later release. You can go ahead with this for now.

**Pull request merged**